features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/metric/RumSessionEndedMetricTest.kt

```diff
@@ -290,7 +290,7 @@ class RumSessionEndedMetricTest {
         // Given
         val sessionEndedMetric =
             SessionEndedMetric(fakeSessionId, fakeStartReason, fakeNtpOffsetAtStart, backgroundEventTracking)
-        val backgroundViewCount = forge.anInt(min = 1, max = events.size)
+        val backgroundViewCount = forge.anInt(min = 1.coerceAtMost(events.size - 1), max = events.size)
```

Comment:
I'm not quite clear with this change: given that `events.size` can be 1 at minimum, it means `min = 0`. Why then simply not specifying `min = 0` directly if we allow this value? 🤔 

But anyway, by looking on this test name we probably want the following: at least 1 background view event among all the view events we receive. It means, given that `max` is exclusive and not inclusive, that the logical change will be simply the following one:

```suggestion
        val backgroundViewCount = forge.anInt(min = 1, max = events.size + 1)
```

If `events.size` is 1, then number generated will be 1 (this is range `[1,2)`). If `events.size` is 2, then generation range will be `[1,3)` = `1,2`.

Reply:
yes, the reason why min=1 is because we have a discussion in the previous PR:
https://github.com/DataDog/dd-sdk-android/pull/2083#discussion_r1642860197

And Indeed I didn't read the doc of  `anInt` carefully,  `max` is actually an exclusive boudary, fixed now.